apps/sms/views/conversation/style/conversation.css

```diff
@@ -157,7 +157,3 @@
 #messages-header-text .ellipsis-dir-fix {
   width: auto;
 }
-
-.conversation-view .panel {
```

Comment:
mmm got you, you're likely right. The rule you're talking about was likely added in the recent bug 1162030. I'll look closer later :)